frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -100,19 +143,22 @@ async function deleteUserRoleAssignment(
                     },
                 }"
             >
-                <TabPanel header="Users">
+                <TabPanel :header="tabHeader">
                     <template #header>
                         <Icon icon="user" :size="IconSize.small" />
                     </template>
                     <UserDataTable
-                        :isApplicationSelected="isApplicationSelected"
+                        v-if="selectedApplicationShortDisplayText == 'FOM_DEV'"
```

Comment:
here as well, this should be the case when the selected application is not FAM, we have many other apps